src/Calculator/GraphingSettings.xaml

```diff
@@ -0,0 +1,136 @@
+<Page
+    x:Class="CalculatorApp.Controls.GraphingSettings"
```

Comment:
I don't think this belongs in the Controls namespace, also can we move the file into the GraphingCalculator folder in addition to the filter?